src/calls-cost.js

```diff
@@ -38,7 +64,23 @@
  * 
  * @returns {CallsResponse}  - Processed information
 */
-
-function callsCost(calls) { }
+    
+function callsCost(calls) {
+  let prices = 0;
+  const details = calls.map((item) => {
+    const price = ProcessedCall(item);
+    prices += price;
+    return price ? {
+      ...item,
+      cost: +price.toFixed(2),
+    } : null;
+  }).filter((item) => item);
```

Comment:
The filter makes sense. But if we think carefully we're iterating the array twice (i.e. once with map plus once with filter).

Could you think of some array method that allows us to make the same process but iterating only once?